File: Two Sum/TwoSum.py
n = int(input())
target = int(input())

nums = list(map(int, input().split()))

def two_sum(nums, target):
    for i in range(len(nums) - 1):
        for j in range(i + 1, len(nums)):
            if nums[i] + nums[j] == target:
                return [i, j]
    return []

ans = two_sum(nums, target)
print(ans)

File: Longest Palindromic Substring/Two Sum/TwoSum.java
import java.util.*;

public class TwoSum {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
        int n = s.nextInt();
        int target = s.nextInt();
        int[] nums = new int[n];
        for (int i = 0; i < n; i++) {
            nums[i] = s.nextInt();
        }
        int[] result = twoSum(nums, target);
        if (result != null) {
            System.out.println(result[0] + " " + result[1]);
        }
    }

    public static int[] twoSum(int[] nums, int target) {
        for (int i = 0; i < nums.length; i++) {
            for (int j = i + 1; j < nums.length; j++) {
                if (nums[i] + nums[j] == target) {
                    return new int[] { i, j };
                }
            }
        }
        return null;
    }
}
